### PR TITLE
OSDOCS-2659 - Updating compliance certifications

### PR DESCRIPTION
**Pull request by pneedle-rh**
This applies to the `dedicated-4` branch only.

This relates to https://issues.redhat.com/browse/OSDOCS-2659. The PR updates the security and control compliance section in the OSD documentation.

The preview is [here](https://deploy-preview-36643--osdocs.netlify.app/openshift-dedicated/latest/osd_policy/policy-process-security?utm_source=github&utm_campaign=bot_dp#compliance_policy-process-security).

### CODE REVIEW

**Comment by pneedle-rh** [comment]
@okashi18 can you please review this from a Product Management perspective? Thanks!

File: modules/policy-security-regulation-compliance.adoc [hunk 1]
```diff
@@ -7,7 +7,7 @@
 = Security and regulation compliance
 
 
-Security and regulation compliance includes tasks such as the implementation of security controls  and compliance certification.
+Security and regulation compliance includes tasks such as the implementation of security controls and compliance certification.
```
Comment (skrthomas) [inline modules/policy-security-regulation-compliance.adoc:10]:
```suggestion
Security and regulation compliance includes tasks, such as the implementation of security controls and compliance certification.
```

**Review by skrthomas — COMMENTED** [review]
Just a missing comma. LGTM!

File: modules/policy-security-regulation-compliance.adoc (hunk 1 above)
Comment (pneedle-rh) [reply modules/policy-security-regulation-compliance.adoc]: @skrthomas thank you for your review! This suggestion is now resolved.

**Comment by pneedle-rh** [comment]
@acathrow can you please review this from a PM perspective? Thanks!

**Comment by pneedle-rh** [comment]
@xueli181114 or @yuwang-RH can you please review this PR from a QE perspective? Thanks!

**Pull request merged**